Makefile

```diff
@@ -0,0 +1,8 @@
+bootstrap:
+	.buildscript/bootstrap.sh
+
+# use yarn scripts for the rest
+%:
+	@yarn $@
+
+.PHONY: bootstrap
```

Comment:
PHONY targets usually suggest improper Makefile usage. It's not Grunt, but instead, a tool for compiling things. I think we're misusing it here.

Reply:
😬 I actually just copied it coz we use it pretty heavily in our Makefiles internally. Happy to remove it here if it doesn't make sense.